examples/imdb_conv.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import absolute_import
+from __future__ import print_function
+import numpy as np
+np.random.seed(1337) # for reproducibility
+
+from keras.preprocessing import sequence
+from keras.optimizers import RMSprop
+from keras.models import Sequential
+from keras.layers.core import Dense, Dropout, Activation, Flatten
+from keras.layers.embeddings import Embedding
+from keras.layers.convolutional import Convolution1D, MaxPooling1D
+from keras.datasets import imdb
+
+# set parameters:
+max_features = 5000
+maxlen = 100
+batch_size = 10
```

Comment:
With such a small batch size you are not utilizing the GPU. I have 95s/epoch with a batch size of 32 on GPU.